### PR TITLE
[DO NOT MERGE] xrpl-py-2.0 changes + bug fixes

### PR DESCRIPTION
**Pull request by connorjchen**
- Fixed bugs like wrong variable names to make tests run
- Changed tests to use xrpl-py-2.0 wallet changes and xrpl-py new alias functions

TODO
- pip install xrpl-py==2.0.0b0 to get new beta changes and test all py files again
- Never tested files in build-a-wallet (need to set up stuff to run wx)
- Never tested key_derivation
- monitor-payments-websocket - read_amount_received no base transaction
- tx-serialization address.py - issues importing local base58
- use-tickets.py - ticket_sequence not valid parameter of AccountSet

### CODE REVIEW

File: content/_code-samples/airgapped-wallet/py/airgapped-wallet-TEST.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def create_wallet(silent: False):
         pub, priv = keypairs.derive_keypair(seed)
         address = keypairs.derive_classic_address(pub)
  
-    return address, seed
```
Comment (mDuo13) [inline content/_code-samples/airgapped-wallet/py/airgapped-wallet-TEST.py:36]: I just made a conflicting change (to lines 132 and 192 which call this function) in #1636 to fix this same problem with the tests. Maybe check that you're branched off the latest version so this doesn't break anything or conflict?

File: content/_code-samples/send-xrp/py/send-xrp.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Example Credentials ----------------------------------------------------------
 from xrpl.wallet import Wallet
-test_wallet = Wallet(seed="sn3nxiW7v8KXzPzAqzyHXbSSKNuN9", sequence=16237283)
+test_wallet = Wallet.from_seed("sn3nxiW7v8KXzPzAqzyHXbSSKNuN9")
```
Comment (mDuo13) [inline content/_code-samples/send-xrp/py/send-xrp.py:3]: Don't you need to specify secp256k1 to get the same address now?

File: content/_code-samples/send-xrp/py/send-xrp.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 print(test_wallet.classic_address) # "rMCcNuTcajgw7YTgBy1sys3b89QqjUrMpH"
```
Comment (mDuo13) [inline content/_code-samples/send-xrp/py/send-xrp.py:4]: I suggest using the new shorter alias here.

```suggestion
print(test_wallet.address) # "rMCcNuTcajgw7YTgBy1sys3b89QqjUrMpH"
```

**Comment by intelliot** [comment]
@JST5000 @mvadari any thoughts on what we should do with this PR?

**Comment by JST5000** [comment]
> @JST5000 @mvadari any thoughts on what we should do with this PR?

Before we fully merge the 2.0 changes we should finish updating the code snippets to follow the new interface. (I think there's one more change that'd be good to add to the 2.0 before merging around updating the steps to submit a transaction to follow the same flow as xrpl.js since it's simpler)

**Comment by mDuo13** [comment]
@connorjchen What's the status of this PR? It's been about 2 months since the last update. I see that xrpl-py 2.0 still hasn't released so maybe this is still on hold waiting for that?

**Comment by connorjchen** [comment]
@mDuo13 My internship ended back in December, unsure what status of PR + xrpl-py is currently

**Comment by JST5000** [comment]
Superseded by https://github.com/XRPLF/xrpl-dev-portal/pull/1933

**Pull request closed without merging**